src/changes.js

```diff
@@ -98,6 +98,21 @@ export async function insertImageFile(
     // case we can quietly prevent updating the image.
     if (!placeholder) return;
 
+    // if there was an error during upload, remove the placeholder image
+    if (props.data.error) {
+      editor.change(change => {
+        change.removeNodeByKey(placeholder.key);
+      });
+
+      if (editor.props.onShowToast) {
+        editor.props.onShowToast("Sorry, an error occurred uploading the image")
+      }
+
+      return;
```

Comment:
I think we can remove the return and use `else` instead. No need to call `onImageUploadStop` separately – it will always be called within the finally block